### ISSUE TITLE
Update for POE 2

### ISSUE THREAD

**Issue by Nicolayka** [SnosMe/awakened-poe-trade#1507]
Hello everyone

Will there be a program update for Path of Exile 2? If so, when should I expect it? Thanks

**Comment by sclient** [comment]
For me, it would be a nice start if I could at least access settings in POE 2 to at least edit macros, and fiddle with cheatsheets and add regex notes to paste for vendor searches.

**Comment by SnosMe** [comment]
Changing this, should give you access to overlay widgets.

![image](https://github.com/user-attachments/assets/091789ba-5338-456b-9da5-b382e95bc97a)

**Comment by Scerutti** [comment]
So, we should modify the “title” and the plugin will work?

**Comment by SnosMe** [comment]
Yes, but only the functionality not related to items.

**Comment by darkracer125** [comment]
i hope someone passionate will take over.  because i can't imagine playing without it.

**Comment by Teamtom1** [comment]
RIP, how to play poe without awakened? I can't go back

**Comment by Runo231** [comment]
pls

**Comment by MikluxoMaklay** [comment]
Игра и так говнище, а теперь еще и без этого божественного аддона

**Comment by Vali-98** [comment]
> p.s. It's not THAT hard to add support, it's hard to support both games at once.

Hey there, was wondering what the workflow is to getting PoE2 working. Is it purely just the parser that needs work?

**Comment by StefanKunde** [comment]
> Yes, but only the functionality not related to items.
> 
> I'm evaluating the decision to support poe2 and leaning towards NOT to. I think it's better to pass the torch to those who are passionate about poe2 and actually play it, lol. 🤔 While I will continue to work on poe1.
> 
> p.s. It's not THAT hard to add support, it's hard to support both games at once.

Would love to do that. But getting those offsets each update is the problem for me. Do you have any ressource on that?

**Comment by Barragek0** [comment]
For anyone interested, there's a fork for poe2 [here](https://github.com/Kvan7/awakened-poe2-trade2)

**Comment by SnosMe** [comment]
I don't want any fork to keep the original name of the app, so please be more creative and change it.
@Kvan7
@pkp24

**Comment by Kvan7** [comment]
Will do, I am at work rn so it will be changed within 12 hours, apologies for the delay on that.

**Comment by pkp24** [comment]
I just wanted something that would open, so if kvan might be properly taking it over I'll just make mine private and use their fork. Sorry for keeping the name too similar

**Comment by Kvan7** [comment]
Noted, yeah already figured out the apis aren't up so that will be kept in mind. 

@SnosMe do you mind if I open a discussion for a question relating to items. It does seem to be, as you mentioned, somewhat east to swap over but I do have a small thing related to RePoE. To try to not clog this repo with stuff for poe2 I can also make it a discussion on the fork and @ you there. Or if you would like to not be asked about it then just let me know too. (Question is relating to normally how you work with it/if you were still using it since it looks unmaintained)

**Comment by SnosMe** [comment]
>small thing related to RePoE
I have private scripts to generate data. But I'd need to write them almost from scratch for poe2, so don't worry about missing them 😅 I also don't use RePoE to export data, and what you are going to use is your choice. 

@Kvan7

**Comment by SnosMe** [comment]
@Kvan7 Yes, using game files, and trade api to match some data, no data from wiki is used.
Scripts are both the secret sauce and the source of pain. Here, a somewhat redacted file, for exporting item names https://gist.github.com/SnosMe/a2c6846be27dd3a9b54dbba46704f44d

**Comment by Kvan7** [comment]
Thank you, that will certainly help a lot! Feel free to take down if you do want it to stay secret secret

**Comment by Kvan7** [comment]
@SnosMe I am leaning towards "Exalted PoE2 Trade", it is still not super creative but is more different than awakened-poe2-trade2. Let me know if you want more more different. I think it is better, but it is inherently the same application, especially right now. But do let me know if it needs to be more different and I will reopen the naming issue. 

(I am not going to update it yet until I get confirmation just since it will move the install and config location so people will have to move stuff again)

**Comment by Barragek0** [comment]
I think 'Exiles Trade Assistant 2' would be a decent name. It doesn't really matter much though, as long as the name gets across that the program is for PoE2 and not PoE1.

**Comment by Nattyx1989** [comment]
> Yes, but only the functionality not related to items.
> 
> I'm evaluating the decision to support poe2 and leaning towards NOT to. I think it's better to pass the torch to those who are passionate about poe2 and actually play it, lol. 🤔 While I will continue to work on poe1.
> 
> p.s. It's not THAT hard to add support, it's hard to support both games at once.

Would love to even just have the functionality of macros for hideout and replys, and checking dps on items etc. Thankyou for all you have done over the years!

**Comment by xAbramx** [comment]
Omg  I cant imagine playing without this tool. I would pay for poe 2 version.

**Comment by Barragek0** [comment]
> Omg I cant imagine playing without this tool. I would pay for poe 2 version.

There's a version being worked on for poe 2 [here](https://github.com/Kvan7/Exiled-Exchange-2), there's a different app called [sidekick](https://sidekick-poe.github.io) (the beta version supports poe 2), or there's poe overlay 2 on overwolf.

**Comment by xAbramx** [comment]
> There's a fork of awakened being worked on for poe 2 [here](https://github.com/Kvan7/Exiled-Exchange-2), there's a different app called [sidekick](https://sidekick-poe.github.io) (the beta version supports poe 2), or there's poe overlay 2 on overwolf.

That is most likely false positive but i scanned this fork with jotti.org and shows 1 issue ( Win.Packed.Mikey-9859574-0).
I have trust issues.

**Comment by Barragek0** [comment]
> > There's a fork of awakened being worked on for poe 2 [here](https://github.com/Kvan7/Exiled-Exchange-2), there's a different app called [sidekick](https://sidekick-poe.github.io) (the beta version supports poe 2), or there's poe overlay 2 on overwolf.
> 
> That is most likely false positive but i scanned this fork with jotti.org and shows 1 issue ( Win.Packed.Mikey-9859574-0). I have trust issues.

It is a false positive, there's a issue on the repo about it.

**Comment by Prof549** [comment]
> 
> I'm evaluating the decision to support poe2 and leaning towards NOT to. I think it's better to pass the torch to those who are passionate about poe2 and actually play it, lol. 🤔 While I will continue to work on poe1.
> 
> p.s. It's not THAT hard to add support, it's hard to support both games at once.

Please, do. The "alternative" is so horrendous(the one containing "wolf" in it's name).
